crates/orderbook/openapi.yml

```diff
@@ -763,7 +769,7 @@ components:
         - sellAmount
         - sellAmountBeforeFees
         - buyAmount
-        - transactionHash
+        - txHash
```

Comment:
Should we mark this as `required`? I was under the impression it isn't always available.

Reply:
It has `nullable: true` and when we mark it as not required it means that it can be `undefined`